### PR TITLE
Climb Stats to Added Map

### PR DESCRIPTION
**Pull request by gibboj**
First step for : https://github.com/OpenBeta/open-tacos/issues/78

Screen Shot 2021-12-27 at 18 07 41

### CODE REVIEW

**Comment by gibboj** [comment]
Do not merge

File: gatsby-node.js [hunk 1]
```diff
@@ -327,3 +337,72 @@ exports.onCreatePage = async ({ page, actions }) => {
...
+      type: String
+      count: Int
+    }
+
+    
+  `;
+  createTypes(typeDefs);
+};
+
+exports.createResolvers = ({ createResolvers }) => {
```
Comment (vnugent) [inline gatsby-node.js:359]: I hope you haven't spent a lot of time on this.  The new GraphQL API is supposed to replace most of complicated graphql code in Gatsby

Comment (gibboj) [reply gatsby-node.js:359]: Nah, plus I wanted to play around with graphQL queries.

Comment (gibboj) [reply gatsby-node.js:359]: Plus I can keep moving forward building UI using that data, then I can figure out how to do it in the new API when you're ready

**Pull request closed without merging**